server/nuxeo-server-tomcat/src/main/assemble/assembly.xml

```diff
@@ -180,6 +180,8 @@
         <include name="log4j-*" /> <!-- log4j2 / bridges: jcl, slf4j, log4j -->
         <include name="json-smart-*" />
         <include name="jsonevent-layout-*" />
+        <!-- install log4j2 JSON logging dependencies in container classpath -->
+        <include name="jackson-*" />
```

Comment:
The two lines preceding the one you introduced also deals with JSON and was added for [NXP-25096](https://jira.nuxeo.com/browse/NXP-25096).

The ticket is mainly about log4j 1 so we should be able to remove these lines.
Note: Link the given ticket to yours as it the same topic just a different logging backend.

Reply:
I added it as related to this issue explaining it was config for log4j1